pages/api/graphql.ts

```diff
@@ -2,18 +2,21 @@ import { ApolloServer, gql } from "apollo-server-micro";
 import { ApolloClient, InMemoryCache } from "@apollo/client";
 import { SchemaLink } from "@apollo/client/link/schema";
 import { makeExecutableSchema } from "@graphql-tools/schema";
-import { getCollectionData, getAllManifests } from "@/services/iiif";
 import slugify from "slugify";
-import { getValues } from "@/hooks/getValues";
-const axios = require("axios");
+
+/**
+ *
+ */
```

Comment:
Again, is there a reason to keep these empty comments? Do we actually want a comment here?

```suggestion
```